src/plugins/data/common/search/session/types.ts

```diff
@@ -0,0 +1,94 @@
+/*
```

Comment:
!NOTE!

Everything in `src/plugins/data/common/search/session/types.ts` just moved from `x-pack/plugins/data/common/search/session/types.ts`

This was needed to improve types and to use `SearchSessionSavedObjectAttributes` type on a client in `src/`. 

Proper `x-pack/plugins/data` clean up will be done very soon in https://github.com/elastic/kibana/issues/92802